packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -214,11 +234,19 @@ class SystemRulesTable extends React.Component {
     calculateParent = ({ profile }, { title, severity, compliant }) => ({
         isOpen: false,
         cells: [
-            title,
-            profile.name,
-            severity,
-            { title: (compliant ? <CheckCircleIcon className='ins-u-passed' /> :
-                <ExclamationCircleIcon className='ins-u-failed' />) }
+            { title, original: title },
```

Comment:
Adding the `original` field here allows us to freely use the `original` to sort, filter, etc.. easily - without it you have to look inside the component, remove the icon.. it's much more of a hassle